README.md

```diff
@@ -56,7 +56,7 @@ See detailed steps for experimenting locally [here](RUNNING_LOCALLY.md).
 - `File > Open` your checkout directory
 - `Run > Edit Configurations`
 - Edit `RegisterRunner`
-- Use classpath of module: `openregister-java_main`
+- Use classpath of module: `openregister-java`
```

Comment:
not sure about this, if I use this classpath I get error:
```
Could not find or load main class uk.gov.register.RegisterApplication
```

Reply:
I'll remove this for now then, and will investigate why it's the other way round for me.